lib/svrquery/protocol/titanfall/query.go

```diff
@@ -174,7 +174,11 @@ func (q *queryer) Query() (resp protocol.Responser, err error) {
 	}
 
 	if i.Version > 2 {
-		if i.Version >= 9 {
+		if i.Version >= 10 {
```

Comment:
Is it worth converting this to a switch statement now the cases are increasing?

Reply:
I do agree it's worth switching over, but I'm going to opt to leave it this time as I've gone through the testing cycle and want to get this delivered shortly. If I had longer I'd look to refactor it a little in general including your recommendation, but I will leave that for when I pick up v11 in the future and start with the refactor if that's ok? 🙂